assets/js/googlesitekit/datastore/user/permissions.js

```diff
@@ -27,6 +27,7 @@ const SET_PERMISSION_SCOPE_ERROR = 'SET_PERMISSION_SCOPE_ERROR';
 
 export const INITIAL_STATE = {
 	permissionError: null,
+	capabilities: global._googlesitekitUserData?.permissions,
```

Comment:
This works, but maybe we should use a resolver like for other data we take from globals? Would be a bit more decoupled.

Reply:
@felixarntz that's what i did at the beginning, but it turned out that resolver didn't work well in this case, so i decided to simplify it and put it straight into initial state.

Reply:
Ok, my bad. It should be good now. Reworked it to use a new resolver.